tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/CopyReferenceOnlyPackages.cs

```diff
@@ -113,7 +113,7 @@ public override bool Execute()
                         if (destination.EndsWith(".nuspec"))
                         {
                             var fileText = File.ReadAllText(destination);
-                            File.WriteAllText(destination, fileText.Replace("</package>", "<files><file src=\".\\**\\*.*\"/></files>\n</package>"));
+                            File.WriteAllText(destination, fileText.Replace("</package>", "<files><file src=\".\\**\\*\"/></files>\n</package>"));
```

Comment:
FYI @dseefeld, this commit explains: https://github.com/dotnet/source-build/pull/916/commits/45efd65430147c26e1c45da6c3bc7a18944f89f3. This only addresses the repack problem, but I think native binaries with no extension are in general being overlooked by this task. I think we should fix in all branches.

Reply:
Good catch.  Yes, I can fix this in the other branches.